admin/packaging.py

```diff
@@ -662,6 +662,16 @@ def __str__(self):
 
         # Cryptography hazmat bindings
         'package-installs-python-pycache-dir opt/flocker/lib/python2.7/site-packages/cryptography/hazmat/bindings/__pycache__/',  # noqa
+
```

Comment:
We are running them in Jenkins (full run) and BuildBot (only new ubuntu wily stuff):
http://ci-live.clusterhq.com:8080/job/ClusterHQ-flocker/job/the-vervet-is-leaving-FLOC-3540/
http://build.staging.clusterhq.com/boxes-flocker?branch=the-vervet-is-leaving-FLOC-3540